packages/neuron-wallet/src/utils/store.ts

```diff
@@ -15,10 +14,17 @@ class Store extends EventEmitter {
 
   constructor(pathname: string, filename: string) {
     super()
+    if (!fs.existsSync(pathname)) {
```

Comment:
Please make sure we remove this after we figure out the loading order issue. This is a potential security issue to create the top level folder.